packages/core/src/external/carequality/dq/create-metadata-xml.ts

```diff
@@ -95,22 +84,35 @@ export function createExtrinsicObjectXml({
     
     <Slot name="sourcePatientId">
       <ValueList>
-        <Value>${patientId}^^^&amp;${homeCommunityId}&amp;ISO</Value>
+        <Value>${patientId}^^^&amp;${METRIPORT_HOME_COMMUNITY_ID_NO_PREFIX}&amp;ISO</Value>
       </ValueList>
     </Slot>
     
     <Name>
       <LocalizedString charset="UTF-8" value="${title}"/>
     </Name>
+
+    <Classification classificationScheme="urn:uuid:93606bcf-9494-43ec-9b4e-a7748d1a838d" classifiedObject="urn:uuid:00000000-0000-d6ba-5161-4e497785491d" id="urn:uuid:953e825d-3907-497c-8a95-bc3761e2a642" nodeRepresentation="" objectType="urn:oasis:names:tc:ebxml-regrep:ObjectType:RegistryObject:Classification">
+      <Slot name="authorPerson">
+        <ValueList>
+          <Value>${organizationName}^^^^^^^&amp;${organizationId}&amp;ISO</Value>
```

Comment:
customer

Reply:
as discussed, this may not be sound logic - https://metriport.slack.com/archives/C04CXLSAF7V/p1711404772669339?thread_ts=1711404046.163589&cid=C04CXLSAF7V